install/example/freebsd_freenas.md

```diff
@@ -1,202 +1,408 @@
 ---
 layout: docs
-title: Installing Airsonic on FreeBSD 10.3 and FreeNAS 9.10
+title: Installing Airsonic on FreeBSD 11.3 and FreeNAS
 permalink: /docs/install/example/freebsd-freenas/
 ---
 #### Preamble
 
-This guide will wallk you through the process of deploying Airsonic on FreeBSD either in a Jail on on the main system. The prerequisites are you have root access on your FreeBSD machine (or jail), the ip address of the machine (or jail) and the Airsonic war available at the [Airsonic github page](https://github.com/airsonic/airsonic/releases).
+This guide will wallk you through the process of deploying Airsonic on FreeBSD either in a Jail on on the main system. Unless you are an advanced user, you will want to install Airsonic inside of a jail. FreeNAS has excellent support for jails, but you can also use a jail manager like [IOcage](https://iocage.io/) The prerequisites are you have root access on your FreeBSD machine (or jail), the ip address of the machine (or jail) and the Airsonic war available at the [Airsonic github page](https://github.com/airsonic/airsonic/releases).
```

Comment:
Please add a link to https://www.freebsd.org/doc/handbook/jails.html on the word Jail :)